Library/Homebrew/cask/audit.rb

```diff
@@ -582,8 +582,9 @@ def check_livecheck_min_os
 
       return if cask_min_os == min_os_string
 
+      min_os_symbol = cask_min_os&.to_sym.inspect || "no minimal OS version"
       add_error "Upstream defined #{min_os_string.to_sym.inspect} as minimal OS version " \
-                "and the cask defined #{cask_min_os.to_sym.inspect}"
+                "and the cask defined #{min_os_symbol}."
```

Comment:
```suggestion
                "and the cask defined #{min_os_symbol}"
```

Sorry, I added the period by accident. I don't think it really matters much bit it didn't have one before